README.md

```diff
@@ -68,8 +64,4 @@ Give a ⭐️ if you like this project!
 - Inspiration
 - etc
```

Comment:
- [ ] Your acknowledgments section in the documentation should contain appropriate credit to Nelson Sakwa whose design was used.